src/test/java/com/ixortalk/organization/api/rest/OrganizationRestResource_Add_IntegrationAndRestDocTest.java

```diff
@@ -111,6 +110,7 @@ public void asAdmin() throws JsonProcessingException {
 
         verify(auth0Roles).addRole(EXPECTED_GENERATED_ROLE_NAME_FOR_MY_TEST_ORGANIZATION);
         verify(auth0Roles, never()).assignRolesToUser(anyString(), anySet());
+        assertThat(restResourcesTransactionalHelper.getUsers("My Organization").size()).isEqualTo(0);
```

Comment:
`assertThat(restResourcesTransactionalHelper.getUsers("My Organization")).isEmpty();` instead of `assertThat(restResourcesTransactionalHelper.getUsers("My Organization").size()).isEqualTo(1);` ?

Reply:
Oh yes, off course, got confused by the `@Before` but it's obviously not yet persisted there.